docs/mobile/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: ONNX Runtime Mobile
```

Comment:
I wanted to keep it consistent with Mobile and Web, and both are prefaced with "ONNX Runtime" currently since we often refer to these as ORT Mobile and ORT Web. Is there a reason to leave off the ONNX Runtime qualifier?

Reply:
I was thinking about it in terms of a neat and consistent nav. It's already clear that it is ORT. And it's in the meta description so will be targeted successfully by search